<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyStadium</class>
 <widget class="QDialog" name="modifyStadium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Stadium</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/img.qrc">
    <normaloff>:/nfl icon.png</normaloff>:/nfl icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Stadium Information</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QLabel" name="labelTypology">
              <property name="text">
               <string>Stadium Typology</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelStadiumName">
              <property name="text">
               <string>Stadium Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="stadiumTypology"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelTeamName">
              <property name="text">
               <string>Team Name</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Date Opened</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="capacity"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="stadiumName"/>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dateOpened"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Seating Capacity</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Grass Surface Type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="teamName"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>League Type</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="grassRadioButton">
                 <property name="text">
                  <string>Grass</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="astroTurfRadioButton">
                 <property name="text">
                  <string>Synthetic</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QRadioButton" name="nationalLeagueRadioButton">
                 <property name="text">
                  <string>National League</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="americanLeagueRadioButton">
                 <property name="text">
                  <string>American League</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>176</height>
            </size>
           </property>
           <property name="title">
            <string>Address and Phone Number</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="1">
             <widget class="QLineEdit" name="address"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="phoneNumber"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelCity">
              <property name="text">
               <string>City</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="state"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="city"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelPhone">
              <property name="text">
               <string>Phone Number</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelAddress">
              <property name="text">
               <string>Street Address</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelState">
              <property name="text">
               <string>State</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="zip"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelZipCode">
              <property name="text">
               <string>Zip Code</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelErrorMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="updateStadiumButton">
       <property name="text">
        <string>Update Stadium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="maximumSize">
        <size>
         <width>480</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>stadiumName</tabstop>
  <tabstop>teamName</tabstop>
  <tabstop>stadiumTypology</tabstop>
  <tabstop>dateOpened</tabstop>
  <tabstop>capacity</tabstop>
  <tabstop>nationalLeagueRadioButton</tabstop>
  <tabstop>americanLeagueRadioButton</tabstop>
  <tabstop>grassRadioButton</tabstop>
  <tabstop>astroTurfRadioButton</tabstop>
  <tabstop>city</tabstop>
  <tabstop>state</tabstop>
  <tabstop>zip</tabstop>
  <tabstop>address</tabstop>
  <tabstop>phoneNumber</tabstop>
  <tabstop>updateStadiumButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../assets/img.qrc"/>
 </resources>
 <connections/>
</ui>
